Django 1.8

sgithens opened the pull request.

sgithens committed:
django-1.8 support. Initial changes to configs and mixin. Not quite ready yet.

Signed-off-by: Steven Githens <swgithen@mtu.edu>

sgithens committed:
django-1.8 Cleaning up test suite

Signed-off-by: Steven Githens <swgithen@mtu.edu>

sgithens committed:
django-1.8 Cleaning up test settings

Removed unused imports and template settings. Also adding comment
on potentially unresolved issue with the INSTALLED_APPS.

Signed-off-by: Steven Githens <swgithen@mtu.edu>

bruth commented:
Please revert the requirements.txt be inclusive of 1.8, not exclusive and update .travis.yml to include Django 1.8.

sgithens committed:
django-1.8 Bumping versions in requirements and travis

Signed-off-by: Steven Githens <swgithen@mtu.edu>

sgithens committed:
Django 1.11

- Moving Status code import from deprecated django module to standard
  location in python 2.7

sgithens committed:
Django 1.11 Work

- Removed url patterns import
- Removed old template context import
- Bumped django version in setup.py

Signed-off-by: Steven Githens <swgithen@mtu.edu>
